blt/src/Blt/Plugin/Commands/MultisiteCommands.php

```diff
@@ -704,14 +704,25 @@ public function create($host, array $options = [
         ->printOutput(FALSE)
         ->printMetadata(FALSE)
         ->run();
+
+      $result = $this->taskGit()
+        ->dir($this->getConfigValue('repo.root'))
+        ->exec('git rev-parse --abbrev-ref HEAD')
+        ->interactive(FALSE)
+        ->printOutput(FALSE)
+        ->printMetadata(FALSE)
+        ->run();
+
+      $branch = $result->getMessage();
+
       $steps += [
-        'Push this branch and merge via a pull request.',
         'Coordinate a new release and deploy to the test and prod environments.',
       ];
     }
 
     $this->say("Committed site <comment>{$host}</comment> code.");
     $this->say("Continue initializing additional multisites or follow the next steps below.");
+    $this->say("Push this branch and merge via a pull request: <comment>git push --set-upstream origin {$branch}</comment>");
```

Comment:
Instead of having this here, can we add it as the first item in `$steps` below?

Reply:
I tried that and it either wouldn't call the branch variable or it would still place it above the steps regardless of where I put it :( If you can figure that out though, I'd be so open to learning how!